src/main/java/SpringBootJava8Startup.java

```diff
@@ -0,0 +1,15 @@
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+@ComponentScan(basePackages = "com.tcs.java8")
+public class SpringBootJava8Startup {
+
+    public static void main(String args[])
```

Comment:
SpringBoot Application is not needed for it. It can be done through maven java project

Reply:
Spring boot application is been created for Learning purpose